src/main/webapp/scripts/itinerary-overview.js

```diff
@@ -79,20 +104,57 @@ function geocodePlaceId(hotelID, placeIDArray) {
         });
     }
 
-    geocoder.geocode({placeId: hotelID}, function(results, status) {
+    geocoder.geocode({ placeId: hotelID }, (results, status) => {
         if (status === "OK") {
             if (results[0]) {
-                hotel += (results[0].geometry.location);
+                console.log("Hotel Address: " + results[0].formatted_address)
+                storeHotelAddress(results[0].formatted_address)
             } else {
                 window.alert("No results found");
```

Comment:
Try to use toasts instead of `alert()`. You can check `my-trips.js` for an example; it should be like `M.toast(...)`